jupyter_console/_version.py

```diff
@@ -1,2 +1,10 @@
-version_info = (5, 0, 0, 'dev')
-__version__ = '.'.join(map(str, version_info))
+""" For beta/alpha/rc releases, the version number for a beta is X.Y.ZbN 
+**without dots between the last 'micro' number and b**. N is the number of
+the beta released i.e. 1, 2, 3 ...
+
+See PEP 440 https://www.python.org/dev/peps/pep-0440/
+"""
+
+version_info = (5, 0, 0, '.dev')
```

Comment:
In general, I think it's a bad idea to do things depending on the version
number, because you don't expect changing a version number to ever affect
how code runs.